.github/workflows/php.yml

```diff
@@ -30,3 +30,45 @@ jobs:
 
       - name: Lint source code
         run: composer phpcs
+
+
+  test:
+    runs-on: ubuntu-22.04
+    
+    strategy:
+      matrix:
+        include:
+          - version: "7.9.0"
```

Comment:
Just for your information, this are the current versions that we use 
    "woocommerce": "8.0.2",
    "composer": "2",
    "wordpress": "6.3.1",
    "php": "7.4"

Reply:
Indeed, I didn't have the latest version of `integration-infrastructure` locally, thanks !